### ISSUE TITLE
Redesign Style /  More advanced default template

### ISSUE THREAD

**Issue by eveneme** [WoltLab/WCF#5811]
A more extensive and functional default template with the option to choose header and menu position templates, especially for mobile devices (top, bottom). The aim is to make the default template more comprehensive and versatile, allowing for customization. Something akin to what will be available in IPS V5. In my opinion, this is a step forward because everyone has individual preferences; some prefer a large header on their computer, while others do not. Please consider this. The Woltlab template is simple and well-optimized, but it lacks the appeal to attract a larger customer base. Please keep in mind that most forum users, especially novices, are primarily influenced by the forum's appearance. The default template should also be refreshed, as it seems to have been created in 2016, which is a considerable amount of time in terms of UI/UX development. It's also important to standardize the appearance of various functions, such as buttons sometimes being displayed across the full width of the screen on mobile devices, while in some cases they are not.

Below, I am sending repositories that can serve as inspiration; they are well-organized in terms of UI/UX.

https://tailwindui.com/components

https://mui.com/components/

Thanks.

**Comment by DracoBluePL** [comment]
I think the best solution would be to have a skilled UI/UX designer and frontend developer create a graphic design using software like Figma or Adobe XD. Developing such a graphic design would greatly facilitate the work during the subsequent template implementation. Creating each layout and standardizing everything to ensure consistency and minimalism. One could say that the new look is a milestone; there's a lot of work involved, but the end result is definitely worth it.

**Comment by cadeyrn** [comment]
> Adobe XD

That's dead software. Maybe you missed it, but Adobe XD is in maintenance mode since almost a year, and in fact they stopped the investment long before that. Even after the failed acquisition of Figma, Adobe confirmed that they have no plans to further invest in XD.

**Comment by DracoBluePL** [comment]
Indeed, thanks for the information. I always used Figma anyway because I liked it better.

**Comment by eveneme** [comment]
I can take care of creating a new design when it comes to graphic design. Contact: info@grafidea.pl

**Comment by eveneme** [comment]
WoltLab has fallen behind in terms of UI/UX. Version 7.0 needs a comprehensive overhaul and a global refresh of its appearance. Unfortunately, many people I’ve spoken to compare WoltLab to phpBB, which, in my opinion, is quite an insult. I have faith in the WoltLab team that in version 7.0, they will thoroughly revamp the look and significantly improve the UI/UX, making the forum experience much more enjoyable.

Just take a look at IPS version 5, they barely introduced any new features or improvements to the code, yet they completely revamped the design. As a result, they received a flood of congratulations and satisfaction from their customers. Let's be honest, the majority of forum users and administrators prioritize appearance, and this is the main factor when choosing a forum engine. That's why, in my opinion, this should be a requirement for WoltLab version 7.0

**Comment by eveneme** [comment]
Many components in WoltLab are currently displayed in a rather raw and basic way, especially considering the capabilities offered by modern styling techniques.

It would be great if, with WoltLab 7.0 and the planned major template overhaul, better-styled and more polished UI components were introduced as well. Modern CSS provides a wide range of powerful styling options, leaving plenty of room for visual and usability improvements.

Additionally, many components could benefit from simplification and more thoughtful styling to ensure a smoother and more enjoyable user experience. WoltLab clearly puts a lot of effort into backend development, which is highly appreciated. It would be great to see more focus placed on UI/UX this year as well.
